pkgs/development/interpreters/python/wrap.sh

```diff
@@ -47,7 +47,7 @@ wrapPythonProgramsIn() {
     buildPythonPath "$pythonPath"
 
     # Find all regular files in the output directory that are executable.
-    for f in $(find "$dir" -type f -perm -0100); do
+    find "$dir" -type f -perm -0100 | while read f; do
```

Comment:
👍 I'm assuming you've tested it with this new implementation? (Just making sure, my memory may not be quite correct on how to do this :)